pages/common/xml-elements.md

```diff
@@ -0,0 +1,28 @@
+# xml elements
+
+> Extract elements and display structure of an XML document.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Extract elements from an XML document (producing XPATH expressions):
+
+`xml elements {{path/to/input.xml|URI}} >{{path/to/elements}}`
+
+- Extract elements and attributes from an XML document:
+
+`xml elements -a {{path/to/input.xml|URI}} >{{path/to/elements}}`
+
+- Extract elements, attributes, and their values from an XML doc:
```

Comment:
```suggestion
- Extract elements, attributes, and their values from an XML document:
```

Reply:
It wrapped when I rendered it, so I trimmed it. I'll make it longer for you.

Reply:
❯ grep "XML doc" xml.md xml-*.md
xml.md:> XMLStarlet Toolkit: Query, edit, check, convert and transform XML documents.
xml-canonic.md:> Make XML documents canonical.
xml-canonic.md:- Make an XML document canonical, preserving comments:
xml-canonic.md:- Make an XML document canonical, removing comments:
xml-edit.md:> Edit an XML document.
xml-elements.md:> Extract elements and display structure of an XML document.
xml-elements.md:- Extract elements from an XML document (producing XPATH expressions):
xml-elements.md:- Extract elements and their attributes from an XML document:
xml-elements.md:- Extract elements and their attributes and values from an XML document:
xml-elements.md:- Print sorted unique elements from an XML document to see its structure:
xml-elements.md:- Print sorted unique elements from an XML document up to a depth of 3:
xml-format.md:> Format an XML document.
xml-format.md:- Format an XML document, indenting with tabs:
xml-format.md:- Recover parsable parts of a malformed XML document, without indenting:
xml-format.md:- Format an XML document from stdin, removing the DOCTYPE declaration:
xml-format.md:- Format an XML document, omitting the XML declaration:
xml-list.md:- List the current directory's contents to an XML document:
xml-list.md:- List the specified directory's contents to an XML document:
xml-pyx.md:> Convert an XML document to PYX (ESIS - ISO 8879) format.
xml-pyx.md:- Convert an XML document to PYX format:
xml-pyx.md:- Convert an XML document from stdin to PYX format:
xml-select.md:> Select from XML documents using XPATHs.
xml-select.md:> Tip: use `xml elements` to display the XPATHs in an XML document.
xml-transform.md:> Transform XML documents using XSLT.
xml-validate.md:> Validate XML documents.
xml-validate.md:- Validate XML document(s) for well-formedness only:
xml-validate.md:- Validate XML document(s) against a Document Type Definition (DTD):
xml-validate.md:- Validate XML document(s) against an XML Schema Definition (XSD):
xml-validate.md:- Validate XML document(s) against a Relax NG schema (RNG):